lang/Português.json

```diff
@@ -134,27 +134,27 @@
         "settings": {
             "general": {
                 "name": "Geral",
-                "autostart": "Auto-Inicio: ",
-                "tray": "Auto-iniciar na bandeija de tarefas: ",
-                "priority_only": "Apenas com Prioridade: ",
-                "proxy": "Proxy (Nessesita Reinicio do App):"
+                "autostart": "Auto-Início: ",
+                "tray": "Auto-iniciar na bandeja de tarefas: ",
```

Comment:
That's not the 'correct' term in Brazilian Portuguese, [according to Microsoft](https://support.microsoft.com/pt-br/windows/personalizar-a-%C3%A1rea-de-notifica%C3%A7%C3%A3o-da-barra-de-tarefas-e159e8d2-9ac5-b2bd-61c5-bb63c1d437c3):

```suggestion
                "tray": "Auto-iniciar na área de notificação: ",
```